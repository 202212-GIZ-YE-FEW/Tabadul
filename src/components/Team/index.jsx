// import React from "react";

// import { TeamCard, TeamDesc, TeamImg, TeamTitle } from "./Team.styled";
// function Team({ Surename, Desctitle, MemberImg }) {
//     return (
//         <TeamCard>
//             <TeamImg src={MemberImg} alt='TeamImg' />
//             <TeamTitle>{Surename}</TeamTitle>
//             <TeamDesc>{Desctitle}</TeamDesc>
//         </TeamCard>
//     );
// }

// export default Team

import React from "react";

function Team() {
    return <div>Team</div>;
}

export default Team;
