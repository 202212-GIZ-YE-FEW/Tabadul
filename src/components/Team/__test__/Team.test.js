import renderer from "react-test-renderer";

import Team from "../index.jsx";

it("renders correctly", () => {
    const tree = renderer.create(<Team />).toJSON();
    expect(tree).toMatchSnapshot();
});
