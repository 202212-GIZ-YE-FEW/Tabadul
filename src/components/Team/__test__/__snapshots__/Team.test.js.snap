// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-wr9924"
>
  <img
    alt="member"
    className="css-1v1g91h"
    data-nimg="1"
    decoding="async"
    height={40}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    property={true}
    src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
    srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
    style={
      Object {
        "color": "transparent",
      }
    }
    width={40}
  />
  <h3
    className="css-1e03y3q"
  >
    Surename Y
  </h3>
  <p
    className="css-1hpa51y"
  >
    Description for member
  </p>
</div>
`;
